### ISSUE TITLE
WLAN Settings integrieren?

### ISSUE THREAD

**Issue by actionbyte** [frickelzeugs/FingerprintDoorbell#96]
Hallo,
ich bin begeistert, tolles Projekt!
Endlich eine tolle Lösung um die Nukis verlässlich zu öffnen ohne auf das Handy warten zu müssen. Danke!
Wäre es möglich die WLAN Settings mit zu integrieren? Im Moment muss man den ESP neu Flaschen um SSID oder sonstiges zu ändern, oder?

**Comment by tilly001** [comment]
Nein. Wenn man den Sensor beim hochfahren berührt, dann wird der AP Point wieder geöffnet.
